From 9ad46071889594123770df4771c36d5b571bfb1b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bernhard=20=C3=9Cbelacker?= <bernhardu@mailbox.org>
Date: Mon, 16 Dec 2024 17:28:09 +0100
Subject: [PATCH 43/86] advapi32: Use no_sanitize_address in
 pointer_from_handle.

provider_from_handle
ret = CryptContextAddRef(0xdeadbeef, NULL, 0);
test_CryptReleaseContext
func_crypt
---
 dlls/advapi32/crypt.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/advapi32/crypt.c b/dlls/advapi32/crypt.c
index 0c94999660a..8444ec7046e 100644
--- a/dlls/advapi32/crypt.c
+++ b/dlls/advapi32/crypt.c
@@ -58,6 +58,7 @@ static HWND crypt_hWindow;
 #define CRYPT_Alloc(size) (LocalAlloc(LMEM_ZEROINIT, size))
 #define CRYPT_Free(buffer) (LocalFree(buffer))
 
+__attribute__((no_sanitize_address))
 static void *pointer_from_handle(UINT_PTR handle, DWORD magic)
 {
     void *ret = NULL;
-- 
2.47.1

